### PR TITLE
London 9 - Mo Mahboobian - JS-Core-1 - Week 1

### PR DESCRIPTION
**Pull request by momahboobian**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Mo Mahboobian
- Your City: London
- Your Slack Name: Mo Mahboobian

# Homework Details

- Module: JS-Core-1
- Week: Week 1

# Notes

- What did you find easy?
 Syntax Errors
- What did you find hard?
Adding decimals
- What do you still not understand?
None
- Any other notes?

### CODE REVIEW

File: mandatory/4-tax.js [hunk 1]
```diff
@@ -17,7 +19,9 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(sales) {
+  return "£" + calculateSalesTax(sales).toFixed(2);
```
Comment (tiritaki) [inline mandatory/4-tax.js:23]: Hi @momahboobian 
I think better practice is to use
`£${result}` - backquote symbols and $ in your return

Comment (momahboobian) [reply mandatory/4-tax.js:23]: Sure, Thank you 😊

**Comment by qingwei91** [comment]
@qingwei91  Please can you review this PR

**Comment by momahboobian** [comment]
@qingwei91 
Please can you review my exersises
Also on extra 03 I had turn off the 1st test, some I can't find where is the problem.
Thanks

File: exercises/K-functions-parameters/exercise.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 // Complete the function so that it takes input parameters
-function multiply() {
+function multiply(num1, num2) {
   // Calculate the result of the function and return it
+  const multiplyNumber = num1 + num2;
```
Comment (qingwei91) [inline exercises/K-functions-parameters/exercise.js:4]: should this be multiply?

File: exercises/K-functions-parameters/exercise2.js [hunk 3]
```diff
@@ -1,5 +1,8 @@
 // Declare your function first
-
+function divide(numA, numB) {
+  const devideNumber = numA - numB;
```
Comment (qingwei91) [inline exercises/K-functions-parameters/exercise2.js:3]: should this be divide?

File: exercises/K-functions-parameters/exercise5.js [hunk 4]
```diff
@@ -1,5 +1,7 @@
 // Declare your function here
-
+function createLongGreeting(name, age) {
+  return `Hello, my name is ${name} + and I'm ${age} years old`;
```
Comment (qingwei91) [inline exercises/K-functions-parameters/exercise5.js:3]: Are you using string interpolation? If so, maybe we don't need the plus sign?

File: mandatory/4-tax.js [hunk 5]
```diff
@@ -5,8 +5,10 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
-
+function calculateSalesTax(sales) {
+  const calcTax = sales * 0.2 + sales;
```
Comment (qingwei91) [inline mandatory/4-tax.js:9]: Can you think of a way to does not requires the +?

**Review by qingwei91 — APPROVED** [review]

File: exercises/K-functions-parameters/exercise2.js (hunk 3 above)
Comment (momahboobian) [reply exercises/K-functions-parameters/exercise2.js]: Yes, sure I made the change 😊

File: exercises/K-functions-parameters/exercise5.js (hunk 4 above)
Comment (momahboobian) [reply exercises/K-functions-parameters/exercise5.js]: Yes, sure I made the change 😊

File: exercises/K-functions-parameters/exercise.js (hunk 2 above)
Comment (momahboobian) [reply exercises/K-functions-parameters/exercise.js]: Yes, sure I made the change 😊

File: mandatory/4-tax.js (hunk 5 above)
Comment (momahboobian) [reply mandatory/4-tax.js:9]: Thank you for making me to find other ways, This is what I like most.
So in this senario we can do this:
`const calcTax = sales * 1.2;`

**Comment by momahboobian** [comment]
@qingwei91 I am receving this conflicts, Can you help me on that please

**Comment by qingwei91** [comment]
Hi @momahboobian , I think we might need to chat over a call.

**Pull request closed without merging**